packages/office-ui-fabric-react/src/components/Icon/Icon.test.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import * as renderer from 'react-test-renderer';
 
 import { Icon } from './index';
+import { TestImages } from 'office-ui-fabric-react/lib/common/TestImages';
```

Comment:
```suggestion
import { TestImages } from '@uifabric/example-data';
```

I think that will fix the issue you're having in the CI.